### DISCUSSION TITLE
Hey, how do i fix this? I'm new to modding and I don't have admin on this laptop, so forge is a no go.

### DISCUSSION THREAD

**Discussion by LightWardenX** [FabricMC/fabric-api#3549] [Mod Dev Support]
### Description

Incompatible mod set!
net.fabricmc.loader.impl.FormattedException: Mod resolution encountered an incompatible mod set!
A potential solution has been determined:
	 - Install forgeconfigapiport, version 3.2.3 or later.
	 - Replace mod 'Fabric API' (fabric) 0.50.0+1.18.2 with version 0.51.1+1.18.2 or later.
	 - Replace mod 'Better Combat' (bettercombat) 1.8.2+1.20.1 with any version that is compatible with:
		 - minecraft 1.18.2
		 - fabricloader 0.13.3
		 - cloth-config 6.2.57
	 - Replace 'Java HotSpot(TM) 64-Bit Server VM' (java) 8 with version 17 or later.
Unmet dependency listing:
	 - Mod 'Fabric API' (fabric) 0.50.0+1.18.2 requires version 17 or later of 'Java HotSpot(TM) 64-Bit Server VM' (java), but only the wrong version is present: 8!
	 - Mod 'Better Combat' (bettercombat) 1.8.2+1.20.1 requires version 1.20.0 or later of 'Minecraft' (minecraft), but only the wrong version is present: 1.18.2!
	 - Mod 'Better Combat' (bettercombat) 1.8.2+1.20.1 requires version 0.14.0 or later of mod 'Fabric Loader' (fabricloader), but only the wrong version is present: 0.13.3!
	 - Mod 'Better Combat' (bettercombat) 1.8.2+1.20.1 requires any version of fabric-api, which is missing!
	 - Mod 'Better Combat' (bettercombat) 1.8.2+1.20.1 requires version 11.0.0 or later of mod 'Cloth Config v6' (cloth-config), but only the wrong version is present: 6.2.57!
	 - Mod 'Better Combat' (bettercombat) 1.8.2+1.20.1 requires version 1.0.0 or later of player-animator, which is missing!
	 - Mod 'YUNG's Better Desert Temples' (betterdeserttemples) 1.18.2-Fabric-1.3.1 requires version 17 or later of 'Java HotSpot(TM) 64-Bit Server VM' (java), but only the wrong version is present: 8!
	 - Mod 'YUNG's Better Dungeons' (betterdungeons) 1.18.2-Fabric-2.1.0 requires version 17 or later of 'Java HotSpot(TM) 64-Bit Server VM' (java), but only the wrong version is present: 8!
	 - Mod 'YUNG's Better Nether Fortresses' (betterfortresses) 1.18.2-Fabric-1.0.0 requires version 17 or later of 'Java HotSpot(TM) 64-Bit Server VM' (java), but only the wrong version is present: 8!
	 - Mod 'YUNG's Better Strongholds' (betterstrongholds) 1.18.2-Fabric-2.1.1 requires version 17 or later of 'Java HotSpot(TM) 64-Bit Server VM' (java), but only the wrong version is present: 8!
	 - Mod 'Farmer's Delight' (farmersdelight) 1.18.2-1.2.5 requires version 0.51.1+1.18.2 or later of mod 'Fabric API' (fabric), but only the wrong version is present: 0.50.0+1.18.2!
	 - Mod 'Geckolib' (geckolib3) 3.0.80 requires version 17 or later of 'Java HotSpot(TM) 64-Bit Server VM' (java), but only the wrong version is present: 8!
	 - Mod 'Biome Particle Weather' (impactfulweather) 4.0.21-1.18.2 requires version 0.51.0 or later of mod 'Fabric API' (fabric), but only the wrong version is present: 0.50.0+1.18.2!
	 - Mod 'Biome Particle Weather' (impactfulweather) 4.0.21-1.18.2 requires version 3.2.3 or later of forgeconfigapiport, which is missing!
	 - Mod 'Biome Particle Weather' (impactfulweather) 4.0.21-1.18.2 requires version 17 or later of 'Java HotSpot(TM) 64-Bit Server VM' (java), but only the wrong version is present: 8!
	 - Mod 'Terrestria' (terrestria) 4.0.6 requires version 17 or later of 'Java HotSpot(TM) 64-Bit Server VM' (java), but only the wrong version is present: 8!
	 - Mod 'TerraBlender' (terrablender) 1.1.0.102 requires version 17 or later of 'Java HotSpot(TM) 64-Bit Server VM' (java), but only the wrong version is present: 8!
	at net.fabricmc.loader.impl.FabricLoaderImpl.load(FabricLoaderImpl.java:190)
	at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:142)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:71)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

**Comment by Linguardium** [comment]
you need to run the game using java 17 not java8

if you download the x64 zip from here you should be able to launch it with the correct binary once set https://adoptium.net/temurin/releases/?version=17&arch=x64&os=windows&package=jdk

**Reply by LightWardenX** [reply]
Okay, I downloaded it. What do I do with it?

**Reply by Linguardium** [reply]
extract it somewhere and set your launcher to use the java.exe from it as the java jvm

**Reply by LightWardenX** [reply]
I'm not using the official launcher, I'm using a jar file.

**Reply by Linguardium** [reply]
then run the jar file from the new java.exe file

`path\to\new\java.exe -jar theJarFile.jar`

**Reply by LightWardenX** [reply]
Okay, so I downloaded and extracted the java 17 zip thingy, where do I look to find what I need? sorry if I'm being frustrating, I've never done anything like this before.

**Reply by Linguardium** [reply]
in the place you extracted it is a bin folder. inside the bin folder is the new java.exe file

you should be running your jar file from that exe using either a shortcut or a bat script

**Reply by LightWardenX** [reply]
Hm, so where do i look for java.exe? Is it in; bin, conf, include or jmods?

**Reply by Linguardium** [reply]
read the last message again

**Reply by LightWardenX** [reply]
how do I do that? do i right click and click open with? im confused
